aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/MergerTest.java

```diff
@@ -139,7 +138,7 @@ public void mergeSaveForNotExistingItem() throws DataStoreException {
             .build();
         ModelMetadata metadata = new ModelMetadata(blogOwner.getId(), false, 1, Time.now());
         // Note that putInStore is NOT called!
```

Comment:
This comment doesn't square with the commented out code.

Reply:
Will fix!